recipes/tachyon/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd unix
+case "$(uname)" in
+    Darwin) target=macosx-x86_64-thr;;
+    Linux) target=linux-64-thr;;
+esac
+
+make $target -j${CPU_COUNT}
+
+cd ..
+cp compile/*/tachyon "$PREFIX/bin"
```

Comment:
Is there no `make install`? If so, maybe worth mentioning to upstream. Are there libraries or headers we need to copy over too?